docs/logging-and-managing-experiment-results/index.rst

```diff
@@ -2,101 +2,153 @@
+
+.. _logging_project:
+
+Project
+^^^^^^^
+It is a **collection of Experiments**, created by user (or users) assigned to the project.
 
-.. |Neptune| raw:: html
+You can log experiments to the project or fetch all experiments that satisfy some criteria.
 
-    <a href="https://neptune.ai" target="_blank">Neptune</a>
+.. code-block:: python3
 
-.. |onboarding| raw:: html
+    # Set project and get project object
+    project = neptune.init('my_workspace/my_project')
 
-    <a href="https://ui.neptune.ai/shared/onboarding/experiments" target="_blank">shared/onboarding</a>
+    # Use project to create experiment
+    project.create_experiment()
 
-.. |github-issues| raw:: html
+    # Use project to get experiments data from the project
+    project.get_leaderboard(state=['succeeded'])
 
-    <a href="https://github.com/neptune-ai/neptune-client/issues" target="_blank">GitHub issues</a>
+Learn more about :ref:`downloading data from Neptune <guides-download_data>`. Check also, :class:`~neptune.projects.Project` to use in your Python code.
 
-.. |Example1| raw:: html
+.. _logging_experiment:
 
-    <a href="https://ui.neptune.ai/USERNAME/example-project/e/HELLO-48/source-code?path=.&file=classification-example.py" target="_blank">Example</a>
+Experiment
+^^^^^^^^^^
+Experiment is everything that you log to Neptune, beginning at ``neptune.create_experiment()`` and ending when script finishes or when you explicitly stop the experiment with :meth:`~neptune.experiments.Experiment.stop`.
 
-.. |Example2| raw:: html
+Creating experiment is easy:
 
-    <a href="https://ui.neptune.ai/USERNAME/example-project/e/HELLO-48/source-code?path=.&file=classification-example.py" target="_blank">Example</a>
+.. code-block:: python3
 
-.. |Example3| raw:: html
+    # Set project
+    neptune.init('my_workspace/my_project')
 
-    <a href="https://ui.neptune.ai/USERNAME/example-project/n/Experiments-analysis-with-Query-API-and-Seaborn-31510158-04e2-47a5-a823-1cd97a0d8fcd/91350522-2b98-482d-bc14-a6ff5c061b6b" target="_blank">Example</a>
+    # Create new experiment
+    neptune.create_experiment()
 
-.. |dashboard| raw:: html
+You can now log various data to the experiment including metrics, losses, model weights, images, predictions and much more. Have a look at the complete list of :ref:`what you can log <what-you-can-log>` to the experiment
 
-    <a href="https://ui.neptune.ai/shared/onboarding/experiments" target="_blank">dashboard</a>
+Besides logging data, you can also :ref:`download experiment data <guides-download_data>` to you local machine or :ref:`update an existing experiment <update-existing-experiment>` even when it's closed.
+
+.. note::
+
+    ``neptune.log_metric('some_name', some_value)`` is for tracking all numeric values to Neptune (metric, loss, score, variances, etc.). Learn, what else can be tracked to experiment from :ref:`this list <what-you-can-log>`.
+
+Learn more about :class:`~neptune.experiments.Experiment` to use in your Python code.
```

Comment:
please reprhase -> perhaps 

Learn more about the Experiment object and how to use it in your Python code.

Reply:
ok